src/Controller/PlanningController.php

```diff
@@ -84,16 +86,8 @@ private function createResponse(Request $request, string $mode): Response
 
         $form->handleRequest($request);
 
-        $service = null;
-
-        if ($form->isSubmitted() && $form->isValid()) {
```

Comment:
I think the $planningData = $this->planningService->getPlanningData(); should only be shown after form is submitted and valid